e2e/test/scenarios/admin/performance/dashboardsAndQuestions.cy.spec.ts

```diff
@@ -47,36 +53,27 @@ const testCacheStrategy = ({
         item: itemToConfigure,
         strategy,
       });
-
-      visitItem();
     });
 
     const expectedCacheDuration = getExpectedCacheDuration(strategy);
+    queryWritableDB(`UPDATE ${TEST_TABLE} SET my_text='Old Value';`);
 
-    advanceServerClockBy(expectedCacheDuration - 2000)
-      .then(() => {
-        reload();
-        wrapResult().as("firstResultBeforeCacheShouldExpire");
-      })
-      .then(() => advanceServerClockBy(1000))
-      .then(() => {
-        reload();
-        wrapResult().as("secondResultBeforeCacheShouldExpire");
-      })
-      .then(() => advanceServerClockBy(5000))
-      .then(() => {
-        reload();
-        wrapResult().as("resultAfterCacheShouldExpire");
-      });
+    // first visit causes us to cache it
+    visitItem();
+    cy.findByTestId("visualization-root").findByText("Old Value");
 
-    cy.then(function () {
-      expect(this.firstResultBeforeCacheShouldExpire).to.eq(
-        this.secondResultBeforeCacheShouldExpire,
-      );
-      expect(this.resultAfterCacheShouldExpire).to.not.eq(
-        this.secondResultBeforeCacheShouldExpire,
-      );
-    });
```

Comment:
I just find this fundamentally confusing. we load, then reload and save a bunch of things and then compare them at the end. Why not just assert on the desired state at each step?